gcn/notices/fermi/gbm/Trigger.example.json

```diff
@@ -6,12 +6,10 @@
     "alert_type": "initial"
   },
   "identifiers": {
-    "instrument_id": "id number",
-    "instrument_id_alternate": "null",
-    "data_archive_page": "null"
-  },
-  "event_info": {
-    "event_id": "Name of the event",
+    "event_id": "id number",
+    "event_id_alternate": "null",
```

Comment:
Why the string “null”?

Reply:
I have no clue! This field is not required, so it should probably just be omitted from the example. I will remove it and test the validation again. Same for the `data_archive_page`